lib/stdlib/js/dummy_threading_semaphore.js

```diff
@@ -0,0 +1,20 @@
+
+function Semaphore(m){
+    this.maxvalue=m;
+    this.value=0;
+    this.handle=128;
```

Comment:
Good catch.  `this.handle` is used in `multiprocessig.synchronize.SemLock`.  I'd hard coded it during testing and forgot to add a mapping of handle ID to Semaphore object.